app/src/sharedTest/java/org/oppia/app/player/exploration/ExplorationActivityTest.kt

```diff
@@ -116,23 +119,33 @@ import javax.inject.Singleton
   qualifiers = "port-xxhdpi"
 )
 class ExplorationActivityTest {
-  private lateinit var networkConnectionUtil: NetworkConnectionUtil
-  private lateinit var explorationDataController: ExplorationDataController
+
+  @Inject
+  lateinit var explorationDataController: ExplorationDataController
```

Comment:
These changes are already there in the develop branch. You need to update your branch with the develop. 
https://github.com/oppia/oppia-android/blob/develop/app/src/sharedTest/java/org/oppia/app/player/exploration/ExplorationActivityTest.kt